src/screens/ProfileScreen/ProfileScreen.js

```diff
@@ -1,21 +1,242 @@
-import React, { Component } from 'react';
-import { View, Text, TouchableOpacity } from 'react-native';
-import styles from './styles';
+import React, { useState, useEffect } from 'react';
+import { View, Text, TextInput, TouchableOpacity, SafeAreaView, Image, ScrollView, Platform } from 'react-native';
 import { firebase } from '../../config/config';
+import ImagePicker from 'react-native-image-crop-picker';
+import styles from './styles';
+
+const ProfileScreen = () => {
+
+    const [name, setName] = useState("")
+    const [occupation, setOccupation] = useState("")
+    const [interest, setInterest] = useState("")
+    const [country, setCountry] = useState("")
+    const [editfield, setEditfield] = useState(false)
+    const [image, setImage] = useState(null)
+    const [displayImage, setDisplayImage] = useState("https://upload.wikimedia.org/wikipedia/commons/thumb/a/ac/No_image_available.svg/600px-No_image_available.svg.png");
```

Comment:
I made changes to Signupform file. I have added one more field in it i.e., "downloadURL" when the User creates an account. This image is required to be in firestore because while implementing other features of the app, I may use this picture only if the user doesn't upload any picture.